ntoskrnl/io/iomgr/iofunc.c

```diff
@@ -4163,7 +4168,8 @@ NtSetVolumeInformationFile(IN HANDLE FileHandle,
     if (PreviousMode != KernelMode)
     {
         /* Validate the information class */
-        if ((FsInformationClass >= FileFsMaximumInformation) ||
+        if ((FsInformationClass < 0) ||
```

Comment:
Same problem here too:
```c
typedef enum _FSINFOCLASS {
  FileFsVolumeInformation = 1,
...
  FileFsMaximumInformation
} FS_INFORMATION_CLASS, *PFS_INFORMATION_CLASS;
```
So better comparing for the min. with `FileFsVolumeInformation` .